src/composables/variants.ts

```diff
@@ -0,0 +1,18 @@
+export const variants = {
```

Comment:
This feels a bit too generally named as there might be components that only need a certain background color for example, but we can tackle that when we run into it.